ui/server/src/main/scala/pl/touk/nussknacker/ui/NussknackerApp.scala

```diff
@@ -35,22 +35,21 @@ import pl.touk.nussknacker.ui.uiresolving.UIProcessResolving
 import pl.touk.nussknacker.ui.validation.ProcessValidation
 import slick.jdbc.{HsqldbProfile, JdbcBackend, PostgresProfile}
 
-
-object NussknackerApp extends App with Directives with LazyLogging {
+trait NussknackerBaseApp extends App with Directives with LazyLogging {
```

Comment:
Let's not extend App here, so we can use this trait not only as standalone.

I think we should also try to use more composition and less inheritance. Can we e.g. think about having a class with core dependencie - like actor system, config and use them to initialize app?

Also: we should probably separate route creation from running, so that sb can embed routes in their own app?